android/src/main/java/com/auth/react/AzureAuthModule.java

```diff
@@ -50,21 +50,10 @@ public String getName() {
 
     @ReactMethod
     public void showUrl(String url, boolean closeOnLoad, Callback callback) {
-        final Activity activity = getCurrentActivity();
-
-        this.callback = callback;
-        this.closeOnLoad = closeOnLoad;
-
-        if (activity != null) {
-            CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
-            CustomTabsIntent customTabsIntent = builder.build();
-            customTabsIntent.launchUrl(activity, Uri.parse(url));
```

Comment:
Actually we do not want to start new browser window if one already started from our app. That's why this if branch. I will add try/catch block to handle possible cashes related to CustomTabsIntent - it could be the case if chrome not installed on the device

Reply:
Sounds good!